src/main/resources/db/changelog/ddl/0010_create_temporary_certificate_tables.xml

```diff
@@ -0,0 +1,121 @@
+            <column name="surname" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="certificate_language" type="varchar(20)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="photo_location_arn" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="issue_date" type="date">
+                <constraints nullable="false"/>
+            </column>
+            <column name="valid_on_date" type="date">
+                <constraints nullable="false"/>
+            </column>
+            <column name="status" type="varchar(50)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="user_id" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="date_created" type="timestamp">
+                <constraints nullable="false"/>
+            </column>
+            <column name="created_by" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="version" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet author="nathan.russell@valtech.com" id="0010-EIP1-3610-create-temporary-certificate-status" context="ddl">
+        <createTable tableName="temporary_certificate_status">
+            <column name="id" type="uuid">
+                <constraints nullable="false" primaryKey="true" unique="true"
+                             primaryKeyName="temporary_certificate_status_pkey"/>
+            </column>
+            <column name="temporary_certificate_id" type="uuid">
+                <constraints nullable="false"
+                             foreignKeyName="fk_temporary_certificate_status_temporary_certificate"
+                             references="temporary_certificate(id)"/>
+            </column>
+            <column name="status" type="varchar(50)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="event_date_time" type="timestamp">
+                <constraints nullable="false"/>
+            </column>
+            <column name="date_created" type="timestamp">
+                <constraints nullable="false"/>
+            </column>
+            <column name="created_by" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="version" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet author="nathan.russell@valtech.com" id="0010-EIP1-3610-source-details-index" context="ddl">
+        <createIndex indexName="source_details_inx" tableName="temporary_certificate" unique="true">
```

Comment:
@NathanRussellValtech , since we can have multiple template certificates for the same `gss_code`, `source_type` and `source_reference`, we will need to have this index as non-unique.

Reply:
Good spot, yes, I'll change that 👍

Reply:
Done

Reply:
Thanks 👍